### ISSUE TITLE
Tautulli connection fails with leading slash in base url

### ISSUE THREAD

**Issue by jgillman** [jorenn92/Maintainerr#1390]
**Describe the bug**
Tautulli connection fails when using a leading slash in base URL.

**To Reproduce**
*(self explanatory, see screenshots)*

**Expected behavior**
I would expect the connection to succeed with or without the leading slash in the base URL.

I'm not sure what the "standard" is but other similar services either automatically add/remove it or display an error. Overseerr, for example, says "URL base must have a leading slash" if it's not included to avoid confusion.

**Screenshots**

Fail with slash
![Fail with slash](https://github.com/user-attachments/assets/d448948a-9f6e-4e15-8566-e06bd3fe4657)

Success without slash
![Success without slash](https://github.com/user-attachments/assets/d7b5d594-d426-4d58-a9f1-b53ac68aeb54)

Nginx logs
![Nginx logs](https://github.com/user-attachments/assets/8cc0f130-71aa-4dd9-95e7-6cd0fd659bec)

**Device:**
 - OS: Slackware 15.0 x86_64 (docker container)
 - Version 2.2.1

**Additional context**
Possibly related to https://github.com/jorenn92/Maintainerr/issues/1305 ?

My screenshots show that I'm connecting with docker networking hostnames but I've confirmed that the behavior is the same if I use a local IP address or external domain.

**Comment by benscobie** [comment]
You're right it could be documented better. I would prefer a leading slash as well. At the very least Tautulli is consistent with the Radarr & Sonarr base URL settings. For now we could just add some help text alongside the input as we've got other higher priority things to work on. A PR is welcome.

**Comment by jgillman** [comment]
Thanks for the quick reply, @benscobie! Agreed that just adding some help text would be sufficient here. I'll see if I can get a PR together.

**Comment by ydkmlt84** [comment]
This is an easy documentation change for me. I will add it to the docs. Maintainerr UI will need a PR though. :)

**Comment by mkaltner** [comment]
Just to add on, I'm having issues getting Maintainerr to talk to my Tautulli that's using HTTPS with self signed certificates.  Nothing in my docker logs, just "Connection failed!" in the UI and a request to /api/settings/test/tautulli with an NOK status.

**Comment by benscobie** [comment]
> Just to add on, I'm having issues getting Maintainerr to talk to my Tautulli that's using HTTPS with self signed certificates. Nothing in my docker logs, just "Connection failed!" in the UI and a request to /api/settings/test/tautulli with an NOK status.

Makes sense as we haven't disabled certificate auth in anyway. Have you got any other apps that talk to services behind a self signed cert? I'm interested how they've implemented a solution.

We could let you specify paths of certs to include for authorisation. You'd need to add them as a mount. We could also provide an environment variable to disable SSL verification.

**Comment by mkaltner** [comment]
I don't think I have anything else talks to Tautulli so I don't have other examples for that specifically.  I generally don't use my reverse DNS when setting things up between eachother and Tautulli doesn't like to run in both HTTP and HTTPS very well.  I've done things like install the certs on my phone and PC to help there.  My certificates are available to all docker containers in the same location so either a docker setting or UI setting would be great, thanks!

**Comment by mkaltner** [comment]
Also, is this related?

`[maintainerr] | 23/01/2025 13:09:27  [INFO] [RuleExecutorService] Not all applications are reachable.. Skipped rule execution.`

There is no `delete` setting for Tautulli (like there is Radarr/Sonarr) so does is Maintainerr erroring out because of that?  I'm not using any Tautulli rules.

**Comment by benscobie** [comment]
> Also, is this related?
> 
> `[maintainerr] | 23/01/2025 13:09:27 [INFO] [RuleExecutorService] Not all applications are reachable.. Skipped rule execution.`
> 
> There is no `delete` setting for Tautulli (like there is Radarr/Sonarr) so does is Maintainerr erroring out because of that? I'm not using any Tautulli rules. All other applications are working correctly via the test button; Plex, Radarr, Sonarr.

Yes, all configured connections are tested regardless of whether they are included in rules or not. See #1416

**Comment by mkaltner** [comment]
I submitted a pull request for this.

![Image](https://github.com/user-attachments/assets/62ae9b58-d371-4a28-a832-aeaec8085e15)

![Image](https://github.com/user-attachments/assets/afade715-cfa4-4b0d-91b2-797a01662646)

![Image](https://github.com/user-attachments/assets/b85f6a8d-79e4-4163-a69c-ad6c8c105df9)

**Comment by mkaltner** [comment]
For now, I used `DB Browser for SQL Lite` to set the tautulli values in settings to `NULL`
